### ISSUE TITLE
Accessibility-specific labels should only exist for alternative to visual / icon UI, and should not duplicate visual info

### ISSUE THREAD

**Issue by danielweck** [edrlab/thorium-reader#601]
https://github.com/readium/readium-desktop/blob/41733d4b5ece3d1810f2972ae107c7b811c1c920/src/resources/locales/en.json#L9

https://github.com/readium/readium-desktop/blob/41733d4b5ece3d1810f2972ae107c7b811c1c920/src/resources/locales/en.json#L78

https://github.com/readium/readium-desktop/blob/41733d4b5ece3d1810f2972ae107c7b811c1c920/src/renderer/components/catalog/PublicationAddButton.tsx#L34-L44

**Comment by danielweck** [comment]
Originating from https://github.com/readium/readium-desktop/issues/600#issuecomment-521925686

=>

I am not sure there should be translated labels specifically for screen reader users. For example, if "book title" is displayed nowhere in the UI for sighted users, then there is no reason why non-sighted users would need to hear this. Conversely, ARIA labelled-by would typically be used for both sighted and VI users, for UI forms / input controls.

https://github.com/readium/readium-desktop/blob/41733d4b5ece3d1810f2972ae107c7b811c1c920/src/resources/locales/en.json#L2-L12

`closeDialog` obeys different rules, because the visual cross is the equivalent, so the text is just an alternative. Same with the `homeMenu` and `searchBook` icons.

`skipLink` is of course fine in the `accessibility` locale bucket.

**Comment by llemeurfr** [comment]
Hi @danielweck can you provide details so that we can move it to actionable requests?

**Comment by danielweck** [comment]
`"bookTitle": "Book title"` has since been removed in the develop branch, so the cleanup process has started via other PRs :)

https://github.com/readium/readium-desktop/blob/799c80d03775f47f81b50b0799f184ef7e82246f/src/resources/locales/en.json#L2-L11

**Comment by danielweck** [comment]
So the remaining duplicate is:

https://github.com/readium/readium-desktop/blob/799c80d03775f47f81b50b0799f184ef7e82246f/src/resources/locales/en.json#L6

https://github.com/readium/readium-desktop/blob/799c80d03775f47f81b50b0799f184ef7e82246f/src/resources/locales/en.json#L76

..used in:
https://github.com/readium/readium-desktop/blob/799c80d03775f47f81b50b0799f184ef7e82246f/src/renderer/components/catalog/PublicationAddButton.tsx#L33-L45

**Comment by danielweck** [comment]
Fixed in recent UI refactors.
